public/script.js

```diff
@@ -106,19 +106,37 @@ function writeEmoji(emoji) {
 
 // For sidebar to work on small screens
 
+function CloseSidebar() {
```

Comment:
@Solai-Adithya everything else looks good but change the first letter of all function names to lowercase, this is the convention we have followed in this repository.

Reply:
Yup! Done